diff --git a/binding.gyp b/binding.gyp
index 6b79e54dfa285ce8d860804e38b5f01d732d0dc4..55b0408280f67d11ff5abb9cf2b21190454353fa 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -42,6 +42,12 @@
                             "AdditionalOptions": ['/std:c++20', '/Zc:__cplusplus']
                         }
                     }
+                }],
+                ['OS=="linux"', {
+                    "libraries": [
+                        "-llz4",
+                        "-lz"
+                    ]
                 }]
             ]
         }
diff --git a/src/gamebryosavegame.cpp b/src/gamebryosavegame.cpp
index 367bed1b1c3cdf685ca6feb8fbb6a62bd3a0a05b..9837beae759b2dd3d3374db3bf9c31e0c4a32429 100644
--- a/src/gamebryosavegame.cpp
+++ b/src/gamebryosavegame.cpp
@@ -21,10 +21,10 @@ uint32_t windowsTicksToEpoch(int64_t windowsTicks)
   return static_cast<uint32_t>(windowsTicks / WINDOWS_TICK - SEC_TO_UNIX_EPOCH);
 }
 
-class MoreInfoException : public std::exception {
+class MoreInfoException : public std::runtime_error {
 public:
   MoreInfoException(const char *message, const char *syscall, const std::string &fileName, int code)
-    : std::exception(std::runtime_error(message))
+    : std::runtime_error(message)
     , m_SysCall(syscall)
     , m_FileName(fileName)
     , m_ErrorCode(code)
