diff --git a/binding.gyp b/binding.gyp
index f73f16d..d8b1865 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -21,9 +21,6 @@
                 "./loot_api/include",
                 "<!(node -p \"require('node-addon-api').include_dir\")"
             ],
-            "libraries": [
-                "-l../loot_api/libloot"
-            ],
             'cflags!': ['-fno-exceptions', '-g', '-O0'],
             'cflags_cc!': ['-fno-exceptions'],
             'msvs_settings': {
@@ -41,6 +38,10 @@
             },
             "conditions": [
               ["OS=='win'", {
+                "libraries": [
+                  "-l../loot_api/libloot",
+                  "-DelayLoad:node.exe"
+                ],
                 "defines!": [
                   "_HAS_EXCEPTIONS=0"
                 ],
@@ -48,9 +49,6 @@
                   "_HAS_EXCEPTIONS=1",
                   "WINVER=0x600"
                 ],
-                "libraries": [
-                  "-DelayLoad:node.exe",
-                ],
                 'msvs_settings': {
                   "VCLibrarianTool": {
                     'AdditionalOptions': [ '/LTCG' ]
@@ -59,6 +57,15 @@
                     'LinkTimeCodeGeneration': 1
                   }
                 }
+              }],
+              ["OS=='linux'", {
+                "libraries": [
+                  "-L../loot_api",
+                  "-llibloot"
+                ],
+                "ldflags": [
+                  "-Wl,-rpath,'$$ORIGIN/../../loot_api'"
+                ]
               }]
             ]
         }
